doc/cephadm/install.rst

```diff
@@ -382,28 +481,40 @@ that configuration isn't already in place (usually in the
 daemons will start up with default settings (e.g., binding to port
 80).
 
-To deploy a set of radosgw daemons for a particular realm and zone::
+To deploy a set of radosgw daemons for a particular realm and zone:
+
+  .. prompt:: bash #
+
+    ceph orch apply rgw *<realm-name>* *<zone-name>* --placement="*<num-daemons>* [*<host1>* ...]"
 
-  # ceph orch apply rgw *<realm-name>* *<zone-name>* --placement="*<num-daemons>* [*<host1>* ...]"
+For example, to deploy 2 rgw daemons serving the *myorg* realm and the *us-east-1* zone on *myhost1* and *myhost2*:
 
-For example, to deploy 2 rgw daemons serving the *myorg* realm and the *us-east-1*
-zone on *myhost1* and *myhost2*::
+  .. prompt:: bash #
```

Comment:
is the indent intended? see https://ceph--37867.org.readthedocs.build/en/37867/cephadm/install/#deploy-rgws .

Reply:
It wasn't. Good catch. This comment inspired me to read through the document again, correcting other similar unintended indentations.

I'll resolve this issue, push and squash, and request your review again, Kefu.